sources/api/apiserver/src/server/error.rs

```diff
@@ -115,6 +115,12 @@ pub enum Error {
 
     #[snafu(display("Unable to send input to config applier: {}", source))]
     ConfigApplierWrite { source: io::Error },
+
+    #[snafu(display("Unable to start shutdown: {}", source))]
+    Shutdown { source: io::Error },
+
+    #[snafu(display("Failed to reboot, exit code: {}, stderr: {:?}", exit_code, stderr))]
```

Comment:
```suggestion
    #[snafu(display("Failed to reboot, exit code: {}, stderr: {}", exit_code, stderr.from_utf8_lossy()))]
```
I think you'd see `Vec[104, 105]` without turning it into a string.